### ISSUE TITLE
[Discussion] Improve/Redo storage of incidents

### ISSUE THREAD

**Issue by alejgh** [Arquisoft/InciManager_i2b#30]
Alright, this is going to be a hard one:
As said in the deliverable, `The system will only store a selection of incidents (those submitted by a person or entity, or some specific values submitted by sensors, not all of them).`, and `Incidents can also have an expiration, for example, in the case of the incidents submitted by a
temperature sensor, if they are submitted each hour, the expiration time will be an hour).`.
In #22 we took the approach of using a JSON configuration file to know which incidents must / must not be saved. We talked  with labra about this, and he said this was not a very elegant solution, so we should think of another way of accomplishing this. 
Right now I can't think of any better way of doing this, so I opened this issue so everyone can tell their ideas and what we should do about this (leave it like it is but implement the expiration date, improve the existing implementation...).

**Comment by ulitol97** [comment]
Did Labra suggest any other viable or more "elegant" solution or give any tip on how to do this better  when he was asked?

**Comment by alejgh** [comment]
No, and we didn't ask him either :sweat_smile:. I think the config file is fine for now, but we should handle at least the expiration time of incidents.

**Comment by ulitol97** [comment]
Now that I'm done deleting CSS bootstrap files and and improving codecov (although some more things could still be done regarding the latter), I wouldn't mind taking care of the expiration time of incidents since most of you have other tasks pending, if you agree with that.
However, I haven´t taken care of storing/parsing incidents so far 😓, so in case you agree and I go ahead with this, any suggestion or guideline on how to implement this (taking for granted that we'll keep the config file)?
